docker/env/version

```diff
@@ -1 +1 @@
-0.66
+0.67-python3.8
```

Comment:
drop python3.8 out of the name,
this is now only for testing, so I can delete those images more clearly